website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -54,3 +67,61 @@ Options:
   -h, --help                           Print help
   -V, --version                        Print version
 ```
+
+### Split DNS
+
+The Linux Client supports Split DNS, which you can enable by setting the
+`FIREZONE_DNS_CONTROL` environment variable to `systemd-resolved` or
+`etc-resolv-conf`. If you don't set this variable, Split DNS will be disabled.
+
+When Split DNS is enabled, the Linux Client will override the system's DNS
+resolver with its internal proxy. To do this, the Linux client needs to know how
+DNS resolution is currently being handled on your system.
+
+#### systemd-resolved
+
+On most modern Linux distributions, DNS resolution is handled by
+`systemd-resolved`. If this is the case for you, set `FIREZONE_DNS_CONTROL` to
+`systemd-resolved`. If you're not sure, you can check by running the following
+command:
+
+```bash
+systemctl status systemd-resolved
+```
+
+#### NetworkManager
+
+In most cases, if you're using NetworkManager your system is likely already
+using `systemd-resolved`, and you should set `FIREZONE_DNS_CONTROL` to
+`systemd-resolved`. You'll need to ensure that `/etc/resolv.conf` is a symlink
+to `/run/systemd/resolve/stub-resolv.conf`:
+
+```bash
+sudo ln -sf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
+```
+
+When NetworkManager detects that symlink exists, it will automatically use
+`systemd-resolved` for DNS resolution.
+
+#### `/etc/resolv.conf`
+
+If you're not using `systemd-resolved`, Firezone supports using the
+`/etc/resolv.conf` file to configure Split DNS as a fallback. To do this, set
+`FIREZONE_DNS_CONTROL` to `etc-resolv-conf`, and the Linux Client will override
+the `/etc/resolv.conf` file with the Firezone internal proxy.
```

Comment:
Added, but is that really the case? I'm guessing it wouldn't have been quite cumbersome to implement.

Reply:
Yeah, there's no unconfigure function here. `ETC_RESOLV_CONF_BACKUP` is only used to create the file here https://github.com/firezone/firezone/blob/afac720d0d2a8b7a0fa43e3dc10615a9d2b66844/rust/connlib/tunnel/src/device_channel/etc_resolv_conf.rs#L18 and then in `main.rs` to read the default resolvers.

We could do it on exit, but since it wouldn't automatically revert on power loss or app crash, it felt weird. I thought I'd do the simple version first

Reply:
Add an `Alert color="warning"` for this one

Reply:
Also see #3817